doc/Install.md

```diff
@@ -47,7 +47,6 @@ For alternative server configurations (such as Nginx server and MariaDB database
 ### Optional 
 
 * PHP ImageMagick extension (php-imagick) for animated GIF support.
-* [Composer](https://getcomposer.org/) for a git install
```

Comment:
Why remove this line?

Reply:
Composer is shipped in the bin directory isn't it?

Reply:
Yes, sorry, I forgot about that.

Reply:
Ok *phew*, your question got me thinking if the `composer.phar` in the directory was only a part of the composer game but the other part had to be installed on the system. And you do know the composer ecosystem better then I do.